src/main/java/net/minecraftforge/registries/RegistryObject.java

```diff
@@ -189,7 +189,8 @@ public void accept(Predicate<ResourceLocation> pred) {
     @Override
     public T get() {
         T ret = this.value;
-        Objects.requireNonNull(ret, () -> "Registry Object not present: " + this.name);
+        if (ret == null)
+            throw new NullPointerException("Registry Object not present: " + this.name);
```

Comment:
Yes, things like this are much cleaner code in my opinion.
It also gives a convenient place to stick a breakpoint if we ever need to debug this NPE.